bazel/mainnet-canisters.bzl

```diff
@@ -0,0 +1,80 @@
+"""Mainnet canister definitions.
+
+This creates a repository which exports 'canister_deps'. This macro can be
+called to create one repository for each canister in the mainnet canister list.
```

Comment:
"create one repository for each canister" — this is a bit confusing to me, could you provide an example?

Reply:
Just for my knowledge. Is whatever goes before the `//` in a target — a Bazel repository? E.g., in `@foo//hello`, `@foo` is the repository?